spec/lib/rbac/filterer_spec.rb

```diff
@@ -843,13 +843,13 @@ def combine_filtered_ids(user_filtered_ids, belongsto_filtered_ids, managed_filt
 
         context "searching MiqTemplate" do
           it "can't see descendant tenant's templates" do
-            owned_template.update_attributes!(:tenant_id => child_tenant.id, :miq_group_id => child_group.id)
+            owned_template.update!!(:tenant_id => child_tenant.id, :miq_group_id => child_group.id)
```

Comment:
Perhaps add another `!` or two here to make sure it really, really updates.

J/K, this is the source of your failures. :)

Reply:
hey there's never enough !!!!s